system/jhtree/jhinplace.cpp

```diff
@@ -269,6 +269,7 @@ inline void skipPacked(const byte * & cur)
 
 //----- special packed format - code is untested
 
+#ifdef ALTERNATIVE_PACKING_FORMAT
```

Comment:
Currently unused, and generates warnings about unused static functions.  Should possibly delete instead.